src/main/java/reo/Parser.java

```diff
@@ -0,0 +1,135 @@
+package reo;
+
+public class Parser {
+    enum Command {
+        BYE,
+        LIST,
+        TODO,
+        EVENT,
+        DEADLINE,
+        MARK,
+        UNMARK,
+        DELETE,
+        FIND,
+        UNKNOWN,
+    }
+    private String input;
+    private String[] words;
+    private Command command;
+    private TaskList tasks;
+    private Storage storage;
```

Comment:
Consider leaving a blank line between your fields and methods for better readability